lib/stats/series.js

```diff
@@ -0,0 +1,209 @@
+import base from 'taskcluster-base';
+
+const BASE_WORKER_SCHEMA = {
+    workerId: base.stats.types.String,
+    workerType: base.stats.types.String,
+    workerGroup: base.stats.types.String,
+    provisionerId: base.stats.types.String
+};
```

Comment:
I suspect we want to add `capacity` here, possibly also `instanceType`... Most cloud providers have a concept of `instanceType`, most even call it `instanceType`. We can just configure the default `instanceType` as `Unknown` if we don't know any.